Aplib.Integrations.Unity/TestRunner.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Aplib.Core;
+using Aplib.Core.Desire.Goals;
+using System;
+using System.Collections;
+
+namespace Aplib.Integrations.Unity
+{
+    /// <summary>
+    /// Represents a test runner for an agent.
+    /// </summary>
+    public class AplibRunner
+    {
+        /// <summary>
+        /// The agent that the test runner is testing.
+        /// </summary>
+        private readonly IAgent _agent;
+
+        public AplibRunner(IAgent agent)
+        {
+            _agent = agent;
+        }
+
+        /// <summary>
+        /// Runs the test for the agent. The test continues until the agent's status is no longer Unfinished.
+        /// </summary>
+        /// <returns>An IEnumerator that can be used to control the execution of the test.</returns>
+        public IEnumerator Test()
+        {
+            while (_agent.Status == CompletionStatus.Unfinished)
+            {
+                // Perform computation or update the agent here
+                _agent.Update();
+
+                // Wait for the next frame
+                yield return null;
```

Comment:
Comment conventions are to end comments with a period, [see Microsoft Docs](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#comment-style).
```suggestion
                // Perform computation or update the agent here.
                _agent.Update();

                // Wait for the next frame.
                yield return null;
```

Reply:
Damn if this ain't nitpicking.

+1 though